src/main/java/CalApp.java

```diff
@@ -0,0 +1,19 @@
+import java.util.Scanner;
+
+import calculator.Calculator;
+
+public class CalApp {
```

Comment:
```suggestion
public class CalculatorApplication {
```

CalApp 만 보아서는 어떤 프로그램인지 알기 어렵습니다.